rflx/model/message.py

```diff
@@ -1632,7 +1676,27 @@ def prune_dangling_fields(
                 break
 
             field, inner_message = inner_messages.pop(0)
-            inner_message = inner_message.prefixed(f"{field.name}_")
+            inner_message, message_attribute_locations = inner_message.prefixed(
+                f"{field.name}_"
+            ).replaced_message_attributes()
+
+            message_next_fields = message.outgoing(field)
+            if len(message_attribute_locations) > 0 and any(
+                n.target != FINAL for n in message_next_fields
+            ):
```

Comment:
I think it is confusing that a list of links is called "fields". `message.outgoing(field)` could be used directly in `any`, the variable is just used once.

Reply:
Changed in 643c841f.